Changes

```diff
@@ -2686,6 +2686,10 @@ OCaml 4.05.0 (13 Jul 2017):
 
 ### Standard library:
 
+- #1229: Add functions to Format module for printing strings conditionally
+  on line breaks.
+  (Josh Berdine)
```

Comment:
@gpetiot you should add yourself here (and @rbonichon and myself as reviewers, I guess)

Reply:
Done.